apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -274,6 +277,14 @@ export function formatProsecutorCourtDateEmailNotification(
         judgeText,
         registrarText: registrarText || 'NONE',
       })
+    : sessionArrangements === SessionArrangements.NONE_PRESENT
```

Comment:
The Contentful string already has logic to deal with session arrangements and it would be better to add this logic there. Besiceds, this will leave the email with an extra newline at the end of the email body.